devdoc/umockcall_requirements.md

```diff
@@ -23,6 +23,8 @@ umockcall is a module that encapsulates a umock call.
     extern int umockcall_get_fail_call(UMOCKCALL_HANDLE umockcall);
     extern int umockcall_set_ignore_all_calls(UMOCKCALL_HANDLE umockcall, int ignore_all_calls);
     extern int umockcall_get_ignore_all_calls(UMOCKCALL_HANDLE umockcall);
+    extern int umockcall_set_call_can_fail(UMOCKCALL_HANDLE umockcall, int call_can_fail);
```

Comment:
>extern [](start = 4, length = 6)

extern not needed. It is by default, so the standard says. Consider removing it for all functions, that's what I do these days when I hit them.